src/Repository/ProductCommentRepository.php

```diff
@@ -26,11 +26,34 @@
 
 namespace PrestaShop\Module\ProductComment\Repository;
 
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
 use Doctrine\DBAL\Connection;
 use Doctrine\DBAL\Query\QueryBuilder;
+use Doctrine\Persistence\ManagerRegistry;
+use Hook;
+use PrestaShop\Module\ProductComment\Entity\ProductComment;
 
-class ProductCommentRepository
+/*
+use Doctrine\DBAL\Cache\QueryCacheProfile;
+use Doctrine\Common\Cache\Psr6\DoctrineProvider;
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+*/
+
+/**
+ * @extends ServiceEntityRepository<ProductComment>
+ *
+ * @method ProductComment|null find($id, $lockMode = null, $lockVersion = null)
+ * @method ProductComment|null findOneBy(array $criteria, array $orderBy = null)
+ * @method ProductComment[] findAll()
+ * @method ProductComment[] findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
```

Comment:
I'm not a fun of such comments, what's the purpose of them?

Reply:
They are generated automatically when you run `Symfony make:entity` in terminal . I think it is useful in case someone try to reinvent the wheel ;)